app/src/schemas/toolCallSchemas.ts

```diff
@@ -0,0 +1,189 @@
+import { z } from "zod";
+import zodToJsonSchema from "zod-to-json-schema";
+
+import { assertUnreachable } from "@phoenix/typeUtils";
+
+/**
+ * The schema for an OpenAI tool call, this is what a message that calls a tool looks like
+ *
+ * Note: The nested passThrough's are used to allow for extra keys in JSON schema, however, they do not actually
+ * allow for extra keys when the zod schema is used for parsing. This is to allow more flexibility for users
+ * to define their own tool calls according
+ */
+export const openAIToolCallSchema = z.object({
```

Comment:
this should replace the one in your playground/toolCallSchemas file above right?